src/bundle/Resources/views/themes/admin/ui/html_body.html.twig

```diff
@@ -0,0 +1,22 @@
+<div class="ibexa-embedded-item-edit-container">
+    {% set form = ibexa_render_embedded_item_edit_form() %}
+
+    {{ form_start(form) }}
+    {{ form_widget(form.content_info, { 'attr': {
+        'hidden': 'hidden',
+        'class': 'ibexa-embedded-item-edit__form-field ibexa-embedded-item-edit__form-field--content-info'
```

Comment:
We need a parent `ibexa-embedded-item-edit` to use such class names so probably this class should be added to the form.

Reply:
https://github.com/ibexa/admin-ui/pull/875/commits/d81cc02eca714a8b476e97d29dce1a420a203032